index.js

```diff
@@ -6,8 +6,8 @@ var colornames = require('colornames');
  * Kuler: Color text using CSS colors
  *
  * @constructor
- * @param {String} text The text that needs to be styled
- * @param {String} color Optional color for alternate API.
+ * @param {string} text The text that needs to be styled
+ * @param {string} [color] Optional color for alternate API.
  * @api public
```

Comment:
Is still understanded? Or does this need to be migrated to `@public` and `@private` as well?